e2e/specs/edit-patient.spec.ts

```diff
@@ -11,7 +11,6 @@ test.beforeEach(async ({ api }) => {
   patient = await generateRandomPatient(api);
 });
 
-// TODO: Add email field after fixing O3-1883 (https://issues.openmrs.org/browse/O3-1883)
```

Comment:
@vasharma05 is it safe to remove this comment? Was [this](https://openmrs.atlassian.net/browse/O3-1883) ticket resolved?

Reply:
It seems, I think we should include the email field to the test.